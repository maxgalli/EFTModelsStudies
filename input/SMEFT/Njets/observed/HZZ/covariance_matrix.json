{
    "r_Njets_0": {
        "r_Njets_0": 0.018863401918270878,
        "r_Njets_1": -0.004898034039949983,
        "r_Njets_2": 0.004369994364253546,
        "r_Njets_3": -0.0008979316637638235,
        "r_Njets_4": -0.004413359782667858
    },
    "r_Njets_1": {
        "r_Njets_0": -0.004898034039949983,
        "r_Njets_1": 0.03982140210985505,
        "r_Njets_2": -0.022338789141291168,
        "r_Njets_3": 0.007166322279170454,
        "r_Njets_4": 0.009020117029613356
    },
    "r_Njets_2": {
        "r_Njets_0": 0.004369994364253546,
        "r_Njets_1": -0.022338789141291168,
        "r_Njets_2": 0.13094101003382577,
        "r_Njets_3": -0.05279022314554644,
        "r_Njets_4": -0.051422173162698925
    },
    "r_Njets_3": {
        "r_Njets_0": -0.0008979316637638235,
        "r_Njets_1": 0.007166322279170454,
        "r_Njets_2": -0.05279022314554644,
        "r_Njets_3": 0.23014090093088133,
        "r_Njets_4": 0.049136574040566615
    },
    "r_Njets_4": {
        "r_Njets_0": -0.004413359782667858,
        "r_Njets_1": 0.009020117029613356,
        "r_Njets_2": -0.051422173162698925,
        "r_Njets_3": 0.049136574040566615,
        "r_Njets_4": 0.7945720302233935
    }
}