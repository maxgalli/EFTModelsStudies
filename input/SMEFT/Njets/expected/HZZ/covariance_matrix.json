{
    "r_Njets_0": {
        "r_Njets_0": 0.01967389787981911,
        "r_Njets_1": -0.00482695208805325,
        "r_Njets_2": 0.003953366043308588,
        "r_Njets_3": -0.000906267358870723,
        "r_Njets_4": 0.0010928616009874687
    },
    "r_Njets_1": {
        "r_Njets_0": -0.00482695208805325,
        "r_Njets_1": 0.05005900987937773,
        "r_Njets_2": -0.02456741189453077,
        "r_Njets_3": 0.018275155643031397,
        "r_Njets_4": 0.0013804324264821252
    },
    "r_Njets_2": {
        "r_Njets_0": 0.003953366043308588,
        "r_Njets_1": -0.02456741189453077,
        "r_Njets_2": 0.1293835095109628,
        "r_Njets_3": -0.1207001577911059,
        "r_Njets_4": 0.035911135110410615
    },
    "r_Njets_3": {
        "r_Njets_0": -0.000906267358870723,
        "r_Njets_1": 0.018275155643031397,
        "r_Njets_2": -0.1207001577911059,
        "r_Njets_3": 0.6997286582697677,
        "r_Njets_4": -0.27596676340905063
    },
    "r_Njets_4": {
        "r_Njets_0": 0.0010928616009874687,
        "r_Njets_1": 0.0013804324264821252,
        "r_Njets_2": 0.035911135110410615,
        "r_Njets_3": -0.27596676340905063,
        "r_Njets_4": 0.7352509529163991
    }
}